src/routes.js

```diff
@@ -1,9 +1,15 @@
 const { Router } = require("express");
+const UserController = require("./controllers/UserController");
 
 const routes = new Router();
 
 routes.get("/", (req, res) => {
   res.json({ message: "Hello World" });
 });
 
+routes.get("/users", (req, res) => UserController.listAll(req, res));
+
+routes.get("/users/:id", (req, res) => UserController.getUserById(req, res));
```

Comment:
Mesmo cenário acima.

```suggestion
routes.get("/users/:id", UserController.getUserById);
```